src/patchelf.cc

```diff
@@ -1833,6 +1839,7 @@ void showHelp(const std::string & progName)
   [--replace-needed LIBRARY NEW_LIBRARY]\n\
   [--print-needed]\n\
   [--no-default-lib]\n\
+  [--no-sort]\t\tDo not sort program+section headers; useful for debugging patchelf.\n\
```

Comment:
Please also mention this flag in the manpage: https://github.com/NixOS/patchelf/blob/master/patchelf.1